docs/ossec_alerts.rst

```diff
@@ -49,7 +49,7 @@ The SMTP relay that you use should support SASL authentication and SMTP
 TLS protocols TLSv1.2, TLSv1.1, and TLSv1. Most enterprise email
 solutions should be able to meet those requirements.
 
-Below are the values you must specify in to configure OSSEC correctly.
+Below are the values you must specify to configure OSSEC correctly.
```

Comment:
This is related to `into` corrections. Here I don't think `in` is valuable

Reply:
I agree, I believe we can omit 'in' entirely here, as you are suggesting.